src/MizarWidgets/SamplingWithFrameTrackWidget.cpp

```diff
@@ -46,6 +48,22 @@ SamplingWithFrameTrackWidget::SamplingWithFrameTrackWidget(QWidget* parent)
   ui_->output_->OnSignificanceLevelChanged(kDefaultSignificanceLevel);
 }
 
+template <typename SendingInput, typename AnotherInput>
+static void ConnectUseItsSymbolToggled(const SendingInput& sender,
+                                       const AnotherInput& another_input,
+                                       bool is_comparison_sending,
```

Comment:
I don't like this variable name because it's a bit confused (comparison might mean several things). Consider use the opposite (`is_baseline_sending`) instead